packages/stark-ui/src/stark-ui.ts

```diff
@@ -1,2 +1,3 @@
 // Export all submodules here
 export * from "./modules";
+export * from "./util/util";
```

Comment:
This barrel is incorrect. Please follow the same approach as the one used for "modules"